src/pages/tutorials/smart-rollups/set-up.md

```diff
@@ -0,0 +1,90 @@
+---
+id: set-up
+title: "Part 1: Setting up the application"
+lastUpdated: 25th October 2023
+---
+
+To set up the application for the tutorial, you must configure Rust to build the kernel and start a Docker container that has resources that are needed to debug and deploy it.
+
+## Building the application
+
+Follow these steps to get the application code and build it:
+
+1. Clone the repository with the tutorial application:
+
+   ```bash
+   git clone https://gitlab.com/trili/hello-world-kernel.git
+   cd hello-world-kernel/
+   ```
+
+1. Configure Rust to build WebAssembly applications:
+
+   1. Verify that you have Rust version 1.73.0 or later installed by running `cargo --version`.
```

Comment:
The install command for Rust is in the prereqs section: 
https://github.com/trilitech/tezos-developer-docs/pull/118/files#diff-ebf65a0487bb3763a4e4bca80dfbe7b7b57559d2d2316e27449612cf6a345559R92
I'll change this line to get the version of rust instead of cargo.